app/gui/view/component-browser/component-list-panel/src/navigator/entry.rs

```diff
@@ -0,0 +1,204 @@
+//! A module containing the grid view entry type which represents an arbitrary icon.
+
+use ensogl_core::prelude::*;
+
+use crate::navigator::icon;
+
+use enso_frp as frp;
+use ensogl_core::application::command::FrpNetworkProvider;
+use ensogl_core::application::frp::API;
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::display;
+use ensogl_core::display::scene::Layer;
+use ensogl_grid_view as grid;
+use ide_view_component_list_panel_icons::Any as AnyIcon;
+use ide_view_component_list_panel_icons::SIZE;
+
+
+
+// ==============
+// === Colors ===
+// ==============
+
+/// Colors of the entry.
+#[derive(Debug, Clone, Copy, Default)]
+pub struct Colors {
+    /// The more contrasting parts of the [icon](crate::icon::Any).
+    pub strong: color::Lcha,
+    /// The less contrasting parts of the [icon](crate::icon::Any).
+    pub weak:   color::Lcha,
```

Comment:
I renamed it to "vivid" and "dull" colors and also I reworked it a little bit in my PR. You will probably have conflicts here. I'd love to chat about it - lets have a short call.

Reply:
Please move it to crate where `AnyIcon` is defined. We can also add a setter getting this as an argument.

Reply:
The struct was removed, as icons use a single color parameter now.